### PR TITLE
style(templates): lint templates

### LINKED ISSUE #216: Some generated files are different than their commited versions
**Describe the bug**
Some files generated by `yo` are different than their versions committed with `husky` hooks. If one wants to pull the bug fixes from a new generator version, it will introduce changes to those files.

I wonder if this has something to do with some `examples` files being reintroduced with every pull request.

**To Reproduce**
1. Generate new files with `yo react-firebase --skip-install`
2. Commit all files with git
3. Regenerate all files with `yo react-firebase --skip-install --force`
4. `git status`

```
On branch master
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   CONTRIBUTING.md
        modified:   README.md
        modified:   database.rules.json
        modified:   firebase.json
        modified:   functions/jsconfig.json
        modified:   jsconfig.json
        modified:   public/manifest.json
        modified:   src/utils/router.js

no changes added to commit (use "git add" and/or "git commit -a")
```

**Expected behavior/code**

```
On branch master
nothing to commit, working tree clean
```

**Possible Solution**

**Additional Context/Screenshots**

### PR DESCRIPTION
**Pull request by mradenovic**
### Description
* Removeslint from template files
* Closes #216

### CODE REVIEW

File: generators/app/templates/jsconfig.json [hunk 1]
```diff
@@ -2,13 +2,8 @@
...
+      "utils/*": ["utils/*"]
     }
   },
-  "exclude": [
-    "node_modules",
-    "dist"
-  ]
-}
\ No newline at end of file
+  "exclude": ["node_modules", "dist"]
```
Comment (prescottprue) [inline generators/app/templates/jsconfig.json:8]: I believe this was shifted originally by prettier, I am wondering if the v2 update is what is now catching this or was it you manually?

**Comment by prescottprue** [comment]
Nice finds! Are you leaving as draft for any particular reason?

File: generators/app/templates/jsconfig.json (hunk 1 above)
Comment (mradenovic) [reply generators/app/templates/jsconfig.json:8]: I did everything manually. `generate --> commit --> re-generate` and everything that shows up as changed should probably be changed in templates too.

**Comment by mradenovic** [comment]
> Are you leaving as draft for any particular reason?

There is more in README, but I didn't dare to touch it without studying it better. Should we merge what's ready?

**Comment by prescottprue** [comment]
Yeah we can get this in then fix the README more later

**Pull request merged**